# Credits to <https://github.com/jpleau>

diff --git a/os/io.c b/os/io.c
index 8181a86..fa024ce 100644
--- a/os/io.c
+++ b/os/io.c
@@ -843,7 +843,7 @@ WriteToClient(ClientPtr who, int count, const void *__buf)
         }
     }   
 #endif
-    if (oco->count == 0 || oco->count + count + padBytes > oco->size) {
+    if (oco->count + count + padBytes > oco->size) {
         FD_CLR(oc->fd, &OutputPending);
         if (!XFD_ANYSET(&OutputPending)) {
             CriticalOutputPending = FALSE;
